ngrep.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: ngrep
+  version: 1.47
+  epoch: 0
+  description: A grep-like utility that allows you to search for network packets on an interface
+  copyright:
+    - license: custom
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - libpcap-dev
+      - pcre-dev
+
+# transform melange version 1.47 => 1_47
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: _
+    to: mangled-package-version
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/jpr5/ngrep/archive/V${{vars.mangled-package-version}}.tar.gz
+      expected-sha512: 47ba65878df6b555701c866721a8a935eabdcce636d398284cbfe5f63baf68c62d994a2f373ea4fc8f44fbed3eecee149f2ee48d39c71c04d34e5a088db8c657
```

Comment:
I think we prefer `git-checkout` whenever we know where to find the source code repository